### ISSUE TITLE
About the veg-fru dataset

### ISSUE THREAD

**Issue by abcdvzz** [HCPLab-SYSU/HSE#1]
Veg-fru dataset has a test set which is much bigger than the train set and val set. I'd like to know if you can tell me which two of them are used as the training set and validation set during training phase. And which one of them is used in your paper to get the top-1 accuracy? Thank you very much.

**Comment by tianshuichen** [comment]
> Veg-fru dataset has a test set which is much bigger than the train set and val set. I'd like to know if you can tell me which two of them are used as the training set and validation set during training phase. And which one of them is used in your paper to get the top-1 accuracy? Thank you very much.

We train our model on the training set, and evaluate it on the test set to get the top-1 accuracy.

**Comment by abcdvzz** [comment]
So: training: train set, validation: test set, top-1 accuracy: test set. However, in the paper of veg-fru, top-1 accuracy is obtained in the val set. So there might be something wrong in your comparison results of veg-fru dataset.

**Comment by tianshuichen** [comment]
We copy the results of HybridNet and CB-CNN from table 5 in the VegFru paper. In the caption, the authors clearly state that the results are evaluated on the test set. It seems nothing is wrong.

**Comment by tianshuichen** [comment]
In fact, we follow the setting of table 5 in the VegFru paper to use train the model on the trainval set, and evaluate it on the test set.

**Comment by Zjay** [comment]
I have tested your model on vegfru. It seems that the model has only 0.8671 top-1 acc.
